content/en-US/api/fontStretch.md

```diff
@@ -0,0 +1,68 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontStretch'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontStretch
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontStretch
+---
+
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/textLetterSpacing](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/textLetterSpacing).**
+
+## Description
+
+The `fontStretch` property of the `CanvasRenderingContext2D` interface
+allows the developers to select different face from a font. The default
+value is `normal`.
+
+## Syntax
+
+`CanvasRenderingContext2D.fontStretch = "ultra-condensed"`
+`CanvasRenderingContext2D.fontStretch = "extra-condensed"` 
+`CanvasRenderingContext2D.fontStretch = "condensed"`
+`CanvasRenderingContext2D.fontStretch = "semi-condensed"` 
+`CanvasRenderingContext2D.fontStretch = "normal"` 
+`CanvasRenderingContext2D.fontStretch = "semi-expanded"`
+`CanvasRenderingContext2D.fontStretch = "expanded"`
+`CanvasRenderingContext2D.fontStretch = "extra-expanded"`
+`CanvasRenderingContext2D.fontStretch = "ultra-expanded"`
+`var fontStretch = CanvasRenderingContext2D.fontStretch`
+
+### Value
+
+A `string` representing current `fontStretch` value. The possible font face values 
```

Comment:
"font face" -> "font-face", I think, feels more natural to me
/shrug

Reply:
updated.

Reply:
It was correct before. Please fix.